cpp/src/Slice/Grammar.y

```diff
@@ -104,17 +104,8 @@ using namespace Slice;
 void
 slice_error(const char* s)
 {
-    // yacc and recent versions of Bison use "syntax error" instead
-    // of "parse error".
-
```

Comment:
The version of Bison that we're using only uses 'syntax error'.
This `if` has probably been dead for years.